components/provisioner/internal/gardener/shoot_reconciler.go

```diff
@@ -143,13 +140,34 @@ func (r *Reconciler) enableAuditLogs(logger logrus.FieldLogger, namespacedName t
 	})
 }
 
-func getAuditLogTenant(seed string, data map[string]string) (string, error) {
+/*
+if can't find exact match for seed (eg. az-us10 key),
+tries to find generic config for seeds group (eg. az-us key)
+*/
+func getAuditLogTenant(seed string, data map[string]string) string {
+	tenant := findTenantStrictly(seed, data)
+	if tenant != "" {
+		return tenant
+	}
+	return findTenantLoosely(seed, data)
+}
+
+func findTenantStrictly(seed string, data map[string]string) string {
+	for key, tenant := range data {
+		if seed == key {
+			return tenant
+		}
+	}
+	return ""
```

Comment:
No need to iterate over the map:

```suggestion
	tenant, found := data[seed]
	if found {
		return tenant
	}
	return ""
```

Reply:
Right, thanks